doc/user/layouts/shortcodes/troubleshooting/disable-user-indexes.html

```diff
@@ -0,0 +1,27 @@
+
+In this mode users...
+
+- Can access objects within the [system catalog][sys-cat] to help determine
+  which indexes are causing the crash.
+- Can only `SELECT` from user-created objects that do not rely on user-created
+  indexes. In essence, this means users can still `SELECT` from user-created...
+  - Tables, but they will never return any data.
+  - Views that contain only references to constant values or depend entirely on
+    system tables' indexes.
+- Cannot `INSERT` data into tables.
+- Can create new objects, but any created indexes are disabled.
+
+After troubleshooting any issues, you can [enable individual
+indexes](/sql/alter-index) or restart Materialize _without_
+disabling user indexes to enable all indexes at once.
+
+For assistance with this mode, see:
+
+- [API Overview: Indexes][api-indexes]
+- [System Catalog][sys-cat]
+- [`SHOW INDEX`](/sql/show-index)
+- [`DROP VIEW`](/sql/drop-view)
+- [`DROP INDEX`](/sql/drop-index)
+
+[api-indexes]: /overview/api-components#indexes
+[sys-cat]: /sql/system-catalog
```

Comment:
How do these links show up? This is an html file, not a md file.

Reply:
You can see it here: https://deploy-preview-9256--materializeinc.netlify.app/cloud/troubleshoot-cloud/#disabling-user-indexes

I checked to make sure our site generator could handle the conversions properly. In this case, I think the process works because the generator inserts the content into the Markdown and then does its Markdown transform.

Reply:
This is great, thanks!